docs/operation/features/ice.md

```diff
@@ -1,30 +1,44 @@
+When on *Loop* main page, tapping on the "Active Carbohydrates" graph will open up the "Carbohydrates" details page that tracks your carb entries for the last 12 hours and how they are absorbed.
+It is helpful to review your meals.
+
 ## Insulin Counteraction Effects
 
-Tapping on your Carbohydrates graph will open up a page that tracks your carb entries for the day (since midnight).  At the top of the page is a useful graphic called Insulin Counteraction Effects (ICE for short).  What is it?  Basically, a graphical representation of ***predicted*** upward-BG effects vs. ***encountered*** upward-BG effects.
+What is *Insulin Counteraction Effects* (*ICE* for short)?
 
-Consider the possible sources of triggers that counteract insulin (in other words, make BGs go up)
+Consider the possible sources of triggers that counteract insulin (in other words, make glucose levels go up)
 
 * food
 * stress
 * illness
 * someone sat too close to you
 
-As we all know, this list can be long but, on "normal" days FOOD will be the single strongest "insulin counteracter".  Food will be the most influence on making BGs go up.  There are also other reasons that BGs climb when we may have expected them to be steady; basals being set too low, failed infusion site, etc.
+As we all know, this list can be long but, on "normal" days food will be the strongest "insulin counteracter".  Food will be the most influence on making glucose levels go up.  There are also other reasons that glucose levels climb when we may have expected them to be steady; basals being set too low, failed infusion site, etc.
+
+If we assume that it's a "normal" day (basals are close to correct, illness is not an issue, the site is good), *Loop* will expect most of the upward pressure on glucose levels are from food.
```

Comment:
When you have carbs on board, Loop *always* assigns ICE to carbs, not just on a normal day. It's probably a good idea to let the user know that this is how Loop looks at it, but it's definitely not always a correct view, and that in situations where you have other positive ICE, like insulin resistance, and carbs on board, that Loop will attribute all the positive ICE to carbs, until all the entered carbs are considered absorbed.  At that point, ICE will start driving RC  upward.

Reply:
If you know that other non-carb effects are affecting your insulin sensitivity significantly (sickness, exercise, etc), all the info about carb absorption should be considered skewed, and not be used for trying to understand meal absorption.